src/main/java/james/altoclef/taskcreator/graphics/PathImporter.java

```diff
@@ -0,0 +1,121 @@
+package james.altoclef.taskcreator.graphics;
+
+import com.intellij.uiDesigner.core.GridConstraints;
+import com.intellij.uiDesigner.core.GridLayoutManager;
+import com.intellij.uiDesigner.core.Spacer;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.*;
+
+public class PathImporter extends JDialog {
+    private JPanel contentPane;
+    private JButton buttonOK;
+    private JButton buttonCancel;
+    private JTextField textField1;
+
+    public PathImporter() {
+        setContentPane(contentPane);
```

Comment:
this is where you would call `setSize`

make sure that if you do this, setVisible(true) is the last item.